### DISCUSSION TITLE
饿了么CDN已无效，将造成默认配置下的Alist显示白屏（附临时解决方案）

### DISCUSSION THREAD

**Discussion by HelimLee** [AlistGo/alist#1430] [General]
自17:30分起，饿了么CDN无效。目前访问饿了么CDN将会返回下列信息
` AccessDenied You have no right to access this object because of bucket acl. 62E4FB27B6FC2031310CDE1E fe-static-zbprod-zb1-oss-3.oss-cn-zhangjiakou.aliyuncs.com `
应对方案：
通过更改CDN为JsDelivr实现恢复访问。
具体操作步骤：
1.通过ssh连接到你的服务器。
2.打开到你安装Alist的目录
3.打开该目录下的data文件夹
4.键入此命令 `netstat -ntlp | grep 5244`，如果您是通过脚本安装的Alist请通过关闭服务的形式先终止Alist服务进程，并跳转第7步
5.观察该命令的返回，并找到诸如"alist"的字样，其后面应跟随一串数字，此为PID（进程ID）
6.输入此命令，`kill -9 进程ID` （注意：这会使Alist服务终止）
7.输入`vi config.json`
8.找到assets键所对应的值，并将其修改为 https://cdn.jsdelivr.net/npm/alist-web@$version/dist/
9.输入`:wq`并回车
10.重新启动Alist

**Comment by wuvei** [comment]
感谢，遇到同样的问题，方法可行。
也可以不用终止进程，直接重启服务：
`systemctl restart alist.service`
